### PR TITLE
Bug 1386459 - Be more careful about variable count limits.

### PR DESCRIPTION
**Pull request by rnewman**

### CODE REVIEW

**Comment by farhanpatel** [comment]
### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
SQLiteLogins.swift | 955 | Limit vertical whitespace to a single empty line. Currently 2. 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Storage/ThirdParty/SwiftData.swift [hunk 1]
```diff
@@ -678,8 +678,9 @@ open class ConcreteSQLiteDBConnection: SQLiteDBConnection {
                 SentryIntegration.shared.sendWithStacktrace(message: "SQLITE_CORRUPT for DB \(filename), \(error)", tag: "SwiftData", severity: .error)
             }
 
-            log.error("SQL error. Error code: \(error.code), \(error) for SQL \(sqlStr).")
-            SentryIntegration.shared.sendWithStacktrace(message: "SQL error. Error code: \(error.code), \(error)", tag: "SwiftData", severity: .error)
+            let message = "SQL error. Error code: \(error.code), \(error) for SQL \(sqlStr.characters.prefix(500))."
```
Comment (st3fan) [inline Storage/ThirdParty/SwiftData.swift]: Could this send PII or will this just contain GUIDs?

File: Storage/ThirdParty/SwiftData.swift [hunk 2]
```diff
@@ -721,8 +722,9 @@ open class ConcreteSQLiteDBConnection: SQLiteDBConnection {
...
```
Comment (st3fan) [inline Storage/ThirdParty/SwiftData.swift]: Same question here about PII.

Comment (rnewman) [inline Storage/ThirdParty/SwiftData.swift]: No; this only sends query strings, which only contain variable placeholders.

**Pull request merged**